internal/flink/command.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/featureflags"
 )
 
+const defaultDatabasePlaceholder = "<current-kafka-cluster>"
```

Comment:
This allows for better documentation. When you type `confluent flink statement create -h` you see that "<current-kafka-cluster>" is the default value instead of "".

Reply:
Oh ok gotcha.